src/RRule.php

```diff
@@ -2018,19 +2018,17 @@ static protected function i18nList(array $array, $and = 'and')
 	 * So region-specific translation files don't need to redefine every strings.
 	 *
 	 * @param string $locale
-	 * @param string $fallbackLocale
 	 *
 	 * @return array
 	 * @throws \InvalidArgumentException
 	 */
-	static protected function i18nLoad($locale, $fallbackLocale = 'en_US')
+	static protected function i18nLoad($locale, $fallback = null)
 	{
-		if ( ! preg_match('/^([a-z]{2})(_[A-Z]{2})?(_[A-Z]*)?$/', $locale, $matches) ) {
-			if ( $fallbackLocale ) {
-				$locale = $fallbackLocale;
-			} else {
-				throw new \InvalidArgumentException('The locale option does not look like a valid locale: '.$locale);
+		if ( ! preg_match('/^([a-z]{2})(_[A-Z]{2})?(?:_[A-Z]*)?/', $locale, $matches) ) {
+			if (!is_null($fallback)) {
+				return self::i18nLoad($fallback);
```

Comment:
I opted to leave in the 'C' locale and instead of having a special case for it - I thought that it should fail here and try the fallback one. I'll fix this in a sec.